storage/entity.go

```diff
@@ -246,7 +247,7 @@ func (e *Entity) MarshalJSON() ([]byte, error) {
 		switch t := v.(type) {
 		case []byte:
 			completeMap[typeKey] = OdataBinary
-			completeMap[k] = string(t)
+			completeMap[k] = t
```

Comment:
This doesn't need to be base-64 encoded?

Reply:
I suppose it already is.

Reply:
t is of type []byte, and golang automatically converts []byte to base64 string when calling json.Marshal().